test/TestProjects/MgmtMockTest/Samples/Generated/DiskEncryptionSets_CreateOrUpdate$$Create a disk encryption set with key vault from a different subscription..cs

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
I am not sure if this file could compile since this file does not have a namespace definition nor a class definition.
But this clearly breaks the format and simplication functionalities in rolsyn -  some global module name is not properly trimmed off (on line 34.
If you are meant to write a file that is using the simple program functionality (I forgot the official name of it) that in a console application, when you want to write a Main function, you could omit the template code just starting from the first line of your program.
But here it might not work, because our test files are not treated as a standalone console application.

Reply:
And the filename contains an extra dot